frontend/src/components/board-components/Gamefield.jsx

```diff
@@ -44,6 +46,15 @@ function Gamefield() {
       newCardOnField: [],
     });
 
+    fetch(`https://api.punkapi.com/v2/beers/random`)
+      .then((response) => response.json())
+      .then((data) => {
+        setEnemyCard(data);
+      })
+      .then(() => {
```

Comment:
This `then()` seems optionnal, you might replace it by a `catch()` to handle possible errors, or just remove it.